fl4health/clients/basic_client.py

```diff
@@ -387,14 +388,32 @@ def _handle_reporting(
         reporting_dict.update(metric_dict)
         self.wandb_reporter.report_metrics(reporting_dict)
 
-    def train_step(self, input: torch.Tensor, target: torch.Tensor) -> Tuple[TrainingLosses, Dict[str, torch.Tensor]]:
+    def _move_data_to_device(self, data: TorchInputType) -> TorchInputType:
```

Comment:
Since this method is restricted to `TorchInputType`, perhaps we call this function, `_move_input_data_to_device` to be specific